src/Models/ProductResult.php

```diff
@@ -21,12 +21,15 @@ class ProductResult
     public ViewedByUserCompanyInfo $viewedByUserCompany;
     public array $filteredVariants;
     public ?HighlightResult $highlight;
+    /** Holds information about how good this product result is. */
+    public Score $score;
     
-    public static function create(string $productId, int $rank) : ProductResult
+    public static function create(string $productId, int $rank, Score $score) : ProductResult
```

Comment:
Technically, a breaking change, but I don't expect that any are actually constructing this, so it is fine that we change it.